src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -529,7 +624,11 @@ else if ( configFileSet.size() == 0 ) {
             }
         }
 
-        if ( !hasGencodeDataSource ) {
+        // Sanity checks to make sure we actually found our data sources:
```

Comment:
Would this imply that there was a folder for a data source + genome build (e.g. `hg38/my_datasource/`), but that it was empty or had invalid files?  If so, that is fine.

Reply:
That is one case, but another would be if there was no gencode directory at all.

If any error in creating the gencode data source occurs, it will not set `hasGencodeDataSource` to true.